modules/wpa_screenshot_capture/src/Plugin/CaptureResponse/ScreenshotCaptureResponse.php

```diff
@@ -96,14 +95,16 @@ private function renderFull(array $options) {
   /**
    * {@inheritdoc}
    */
-  public static function compare(CaptureResponseInterface $a, CaptureResponseInterface $b) {
-    $size1 = $a->getCaptureSize();
-    $size2 = $b->getCaptureSize();
-    $variance = 100 * abs($size2 - $size1) / $size1;
-    $response = new ScreenshotVarianceCompareResponse($variance);
-    $response->setFile1Size($size1);
-    $response->setFile2Size($size2);
-    return $response;
+  public static function compare(CaptureResponseInterface $a, CaptureResponseInterface $b, array $compare_utilities) {
+    $comparison_utility_manager = \Drupal::service('plugin.manager.comparison_utility');
+    $response_factory = \Drupal::service('web_page_archive.compare.response');
+    $response_collection = $response_factory->getCompareResponseCollection();
+    foreach ($compare_utilities as $compare_utility) {
+      $instance = $comparison_utility_manager->createInstance($compare_utility);
+      $comparison_response = $instance->compare($a, $b);
+      $response_collection->addResponse($comparison_response);
+    }
+    return $response_collection;
```

Comment:
This code should live in the base class, but I don't want to have to rewrite the code for the other capture utilities as part of this PR as it's already pretty large as it is. So I'll create a separate issue for that.